CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # CHANGELOG
 
+## 4.5.00
+
+[Full Changelog](https://github.com/kokkos/kokkos/compare/4.0.00...4.5.0.)
+
+### Features:
+* Introduce new `SequentialHostInit` view allocation property [\#7229](https://github.com/kokkos/kokkos/pull/7229)
+* Support building with Run-Time Type Information (RTTI) disabled
+* Add `KOKKOS_RELOCATABLE_FUNCTION` [\#5993](https://github.com/kokkos/kokkos/pull/5993)
+
+### Backend and Architecture Enhancements:
+
+#### CUDA:
+* Adding occupancy tuning for CUDA architectures [\#6788](https://github.com/kokkos/kokks/pull/6788) 
+* Add support for CUDA unified memory architectures, e.g.,  Grace Hopper [\#6823](https://github.com/kokkos/kokks/pull/6823)
+* By default disable `cudaMallocAsync`, i.e., reverts the change made in version 4.2 [\#7353](https://github.com/kokkos/kokkos/pull/7353)
+
+#### HIP:
+* Add support for AMD Phoenix APUs with Radeon 740M/760M/780M/880M/890M [\#7162](https://github.com/kokkos/kokkos/pull/7162)
+* Update maximum waves per CU values for consumer card [\#7347](https://github.com/kokkos/kokkos/pull/7347)
+* Fix compilation of global launch for graph nodes [\#7365](https://github.com/kokkos/kokkos/pull/7365)
+* Check that Kokkos is running on the architecture it was compiled for [\#7379](https://github.com/kokkos/kokkos/pull/7379)
+* Add opt-in option to use `hipMallocAsync` instead of `hipMalloc` [\#7324](https://github.com/kokkos/kokkos/pull/7324)
+* Introduce new architecture flag `AMD_GFX942_APU` for MI300A [\#7462](https://github.com/kokkos/kokkos/pull/7462)
+
+#### SYCL:
+* Move the `SYCL` backend out of the `Experimental` namespace [\#7171](https://github.com/kokkos/kokkos/pull/7171)
+* Introduce KOKKOS_ENABLE_SYCL_RELOCATABLE_DEVICE_CODE as CMake option [\#5993](https://github.com/kokkos/kokkos/pull/5993) 
+
+#### OpenMPTarget:
+* Refactor the code to centralize tag evaluation [\#7200](https://github.com/kokkos/kokkos/pull/7200) [\#7211](https://github.com/kokkos/kokkos/pull/7211)
+* Refactor the code to separate DeepCopy implementation [\#7192](https://github.com/kokkos/kokkos/pull/7192)
+* Block tests that fail with the CrayClang compiler [\#7355](https://github.com/kokkos/kokkos/pull/7355)
+#### OpenACC:
+* Update cmake/make 1) to compile the OpenACC backend using Clacc and  2) for the OpenACC backend to use CUDA/HIP runtime to calculate the concurrency [\#7198](https://github.com/kokkos/kokkos/pull/7198)
```

Comment:
I would spit this in two entries. We could also just drop the second part since it's mostly not visible to the user.

Reply:
Yeah, I thought so.